app/views/curation_concerns/base/show.html.erb

```diff
@@ -22,8 +26,11 @@
 <div class="row">
   <div class="col-sm-12">
     <%= render 'items', presenter: @presenter %>
+    <%= render 'workflow_actions_widget', presenter: @presenter %>
```

Comment:
Can we move the `FlipFlop` check out of the `workflow_actions_widget` partial and into here?  This will keep the program for having to find a partial that doesn't do anything.